docs/running-on-yarn.md

```diff
@@ -48,9 +48,9 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
 </tr>
 <tr>
   <td><code>spark.yarn.am.waitTime</code></td>
-  <td>100000</td>
+  <td>100s</td>
```

Comment:
unfortunately this needs to be `100000ms` for backward compatibility, because we stated in the docs that these configs accept raw numeric values assuming the units of the default value.